tools/Equinox.Tool/Program.fs

```diff
@@ -314,11 +325,19 @@ module CosmosInit =
     let containerAndOrDb log (iargs: ParseResults<InitArguments>) = async {
         match iargs.TryGetSubCommand() with
         | Some (InitArguments.Cosmos sargs) ->
-            let rus, skipStoredProc = iargs.GetResult(InitArguments.Rus), iargs.Contains InitArguments.SkipStoredProc
-            let mode = if iargs.Contains InitArguments.Shared then Provisioning.Database rus else Provisioning.Container rus
-            let modeStr, rus = match mode with Provisioning.Container rus -> "Container",rus | Provisioning.Database rus -> "Database",rus
+            let skipStoredProc = iargs.Contains InitArguments.SkipStoredProc
             let client,dName,cName = conn log sargs
-            log.Information("Provisioning `Equinox.CosmosStore` Store at {mode:l} level for {rus:n0} RU/s", modeStr, rus)
+            let mode = (CosmosInitInfo iargs).ProvisioningMode
+            match mode with
```

Comment:
Arguably, this logging code could move up into `CosmosInitInfo` - splitting parsing/logging from the doing, but in general in the tool, I don't follow [the rules the templates do](https://github.com/jet/dotnet-templates#module-args) to the same degree